extensions/wikia/UserPreferencesV2/UserPreferencesV2.i18n.php

```diff
@@ -15,19 +15,18 @@
 	'preferences-v2-date' => 'Date format:',
 	'preferences-v2-time' => 'Time zone:',
 	'preferences-v2-skin' => 'Layout:',
-
 	'prefs-emailv2' => 'Email',
 	'prefs-email-me-v2' => 'Email me when...',
 	'tog-enotifwatchlistpages-v2' => "...a page I'm following is changed",
 	'tog-enotifusertalkpages-v2' => '...my user talk page is changed',
 	'tog-enotifminoredits-v2' => "...a page I'm following is minorly changed",
+	'tog-enotifdiscussions-v2' => '...my Discussion posts receive replies and upvotes',
```

Comment:
This and the qqq (line 85) were the only addition for this ticket. The rest of the changes were cherry-picked from https://github.com/Wikia/app/pull/10741/files#diff-9cfccc6e1a2134c92bc3ae75aa33fbe1L15